sema/commons/glob/__init__.py

```
@@ -0,0 +1,13 @@
+    GlobMatchVisitor,
+    getMatchingGlobPaths,
+    pathMatchesGlob,
+    visitGlobPaths,
+)
+
+__all__ = [
+    "getMatchingGlobPaths",
+    "pathMatchesGlob",
+    "GlobMatchVisitor",
+    "visitGlobPaths",
+]
```

Ruff (0.8.2)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)